samples/bluetooth/peripheral_with_multiple_identities/sample.yaml

```diff
@@ -0,0 +1,16 @@
+sample:
+  description: Peripheral with multiple identities sample
+  name: Peripheral with multiple identities
+tests:
+  sample.bluetooth.scanning_while_connecting:
+    sysbuild: true
+    build_only: true
+    integration_platforms:
+      - nrf52dk/nrf52832
+      - nrf52840dk/nrf52840
+      - nrf5340dk/nrf5340/cpunet
+      - nrf54l15dk/nrf54l15/cpuapp
```

Comment:
Can you also add `nrf54l15dk/nrf54l10/cpuapp` and `nrf54l15dk/nrf54l05/cpuapp` to both `integration_platforms` and `platform_allow`? You may have to rebase on main to get it working.

Reply:
Have rebased and made the change now :) Are these platforms all correct? Do I need to try build for 54l10 or is this done in CI?

Reply:
It will be done automatically :)